_develop/mining_enroll.md

```diff
@@ -17,74 +17,87 @@ rewards. This section explains the enrollment process and covers the following t
 
 ## Complete the add and submission forms on App.co
```

Comment:
Since this section includes information only about the _one_ form shown on the "Submit app" page, would it not be more correct to write this header as simply "Complete the app submission form on App.co"?

Reply:
Yep, I'll fix that.